pkg/brpb/brpb.pb.go

```diff
@@ -61,7 +61,7 @@ func (x CompressionType) String() string {
 	return proto.EnumName(CompressionType_name, int32(x))
 }
 func (CompressionType) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_brpb_a90a70ff0dc32c8e, []int{0}
+	return fileDescriptor_brpb_7c3d2f8143177a80, []int{0}
```

Comment:
would you like to reset the unrelated files?

Reply:
These differences are generated by running `make go`.

Reply:
I know. You can checkout unrelated files to make the diff smaller.

Reply:
got it.